docs/Feature_guides/Real-time_updates.md

```diff
@@ -0,0 +1,188 @@
+## Introduction
+
+This guide goes over the implementation of "real-time updates" using GraphQL subscriptions.
+
+What it means is that when a client(browser) goes to a page with a **subscription** it get subscribed to it using a **websocket**. Javascript is then listening to any message coming through this **subscription** and can change the page content based on it.
+
+We take our implementation of product backlog board as an example, which, to simplify, displays multiple columns of ordered cards.\
+Let's say multiple users are looking at the same board, moving cards around. Their "update" requests are sent to the server, as mutations if using graphql. If they're **successful** they trigger subscriptions and the changes are **broadcast** to every subscriber. That way everyone sees the same state of cards without needing to refresh the page. That's what we call "real-time updates".
+
+The guide is divided into front-end(FE) and back-end(BE) sections and describes requirements, implementation and recommended _PATTERNS_ used.
+
+## Requirements
+
+### Front-end
+
+- `apollo-client`
+- `apollo-link-http`
+- `graphql-ruby-client/subscriptions/ActionCableLink`
+
+Main configuration is done in apollo client - `app/frontend/src/shared/plugins/apolloClient.js`
+
+### Back-end
+
+- `gem 'graphql'`
+- working websockets (one is enough) -> GraphQL subscriptions use [ActionCable](https://guides.rubyonrails.org/action_cable_overview.html) internally so `cable.yml` needs to be configured
+
+## Implementation
+
+_PATTERN:_ Use multiple context-oriented subscriptions instead of a single big one. They're easier to manage and customize avoiding `if` conditions.\
+In our backlog board example we use 5 different subscriptions:
```

Comment:
So that subscriptions for individual boards do not conflict. We use subscription_scope.

Reply:
After testing, subscription_scope will be removed so we leave it out.